### DISCUSSION TITLE
FullCalendar to Nextjs application

### DISCUSSION THREAD

**Discussion by KristjanVall** [vercel/next.js#21519] [Help] [answered]
Hello

how can I add FullCalendar to my Next.js application. 
I have searched in google, but all the tutorials are not working.

Would any has a GitHub example or some working examples (tutorials) :)

Thx

**Comment by jamesmosier** [comment, marked as answer]
Hi @KristjanVall. Take a look at the fullcalendar-example repo. I added a PR that shows an implementation https://github.com/fullcalendar/fullcalendar-example-projects/pull/16/files

**Reply by KristjanVall** [reply]
<img width="1266" alt="Screenshot 2021-01-26 at 01 53 50" src="https://user-images.githubusercontent.com/33087780/105780559-65e32a80-5f79-11eb-83cf-536df0b6bb88.png">

When i run the https://github.com/fullcalendar/fullcalendar-example-projects/tree/master/next-scheduler npm run watch i got this error (in picture). 

Also i use Tailwind and it's uses postcss, but i do not understand how can i run babel and postcss together?

**Reply by jamesmosier** [reply]
That example is incorrect. The preset should be `next/babel` and not `'@babel/preset-react'`

And take a look at the [tailwind](https://github.com/vercel/next.js/tree/canary/examples/with-tailwindcss) example for how to setup postcss and tailwind.
